src/components/PendingTxsSnackbar/store.test.js

```diff
@@ -0,0 +1,317 @@
+      await store.queryPendingTransactions();
+      expect(store.count).toBe(10);
+      expect(store.pendingCreateEvents.length).toBe(2);
+      expect(store.pendingBets.length).toBe(1);
+      expect(store.pendingSetResults.length).toBe(2);
+      expect(store.pendingVotes.length).toBe(2);
+      expect(store.pendingFinalizeResults.length).toBe(1);
+      expect(store.pendingWithdraws.length).toBe(2);
+      expect(store.pendingTransfers.length).toBe(0);
+      expect(store.pendingResetApproves.length).toBe(0);
+      expect(store.isVisible).toBe(true);
+
+      mockAddTransaction({ txid: 11, status: TransactionStatus.PENDING, type: TransactionType.TRANSFER });
+      await store.queryPendingTransactions();
+      expect(store.count).toBe(11);
+      expect(store.pendingCreateEvents.length).toBe(2);
+      expect(store.pendingBets.length).toBe(1);
+      expect(store.pendingSetResults.length).toBe(2);
+      expect(store.pendingVotes.length).toBe(2);
+      expect(store.pendingFinalizeResults.length).toBe(1);
+      expect(store.pendingWithdraws.length).toBe(2);
+      expect(store.pendingTransfers.length).toBe(1);
+      expect(store.pendingResetApproves.length).toBe(0);
+      expect(store.isVisible).toBe(true);
+
+      mockAddTransaction({ txid: 12, status: TransactionStatus.PENDING, type: TransactionType.RESET_APPROVE });
+      await store.queryPendingTransactions();
+      expect(store.count).toBe(12);
+      expect(store.pendingCreateEvents.length).toBe(2);
+      expect(store.pendingBets.length).toBe(1);
+      expect(store.pendingSetResults.length).toBe(2);
+      expect(store.pendingVotes.length).toBe(2);
+      expect(store.pendingFinalizeResults.length).toBe(1);
+      expect(store.pendingWithdraws.length).toBe(2);
+      expect(store.pendingTransfers.length).toBe(1);
+      expect(store.pendingResetApproves.length).toBe(1);
+      expect(store.isVisible).toBe(true);
+
+      mockResetTransactionList();
+      await store.queryPendingTransactions();
+      expect(store.count).toBe(0);
+      expect(store.pendingCreateEvents.length).toBe(0);
+      expect(store.pendingBets.length).toBe(0);
+      expect(store.pendingSetResults.length).toBe(0);
+      expect(store.pendingVotes.length).toBe(0);
+      expect(store.pendingFinalizeResults.length).toBe(0);
+      expect(store.pendingWithdraws.length).toBe(0);
+      expect(store.pendingTransfers.length).toBe(0);
+      expect(store.pendingResetApproves.length).toBe(0);
+      expect(store.isVisible).toBe(false);
+
+      mockAddTransaction({ txid: 4, status: TransactionStatus.PENDING, type: TransactionType.SET_RESULT });
+      await store.queryPendingTransactions();
+      expect(store.count).toBe(1);
+      expect(store.pendingCreateEvents.length).toBe(0);
+      expect(store.pendingSetResults.length).toBe(1);
+      expect(store.isVisible).toBe(true);
+    });
+
+    it('Fetches all the pending txs and increments the counts', async () => {
```

Comment:
duplicate test? this looks like the first test. delete if its testing the same thing.